--- libraw1394-2.0.5/doc/libraw1394.sgml.orig	2009-12-26 17:01:39.000000000 +0100
+++ libraw1394-2.0.5/doc/libraw1394.sgml	2010-06-23 07:08:51.411448126 +0200
@@ -1,6 +1,4 @@
-<!--
-<!DOCTYPE book PUBLIC "-//OASIS//DTD DocBook V3.1//EN">
--->
+<!DOCTYPE book PUBLIC "-//OASIS//DTD DocBook V4.1//EN">
 <book>
   <bookinfo>
 
@@ -3231,7 +3229,7 @@
 </refsect1>
 </refentry>
 
-<refentry id="API-int">
+<refentry id="API-arm-req-callback-t">
 <refentryinfo>
  <title>LINUX</title>
  <productname>Kernel Hackers Manual</productname>
