--- libraw1394-0.9.0/libraw1394.m4.orig	Mon May 14 02:56:29 2001
+++ libraw1394-0.9.0/libraw1394.m4	Fri Dec 19 12:48:53 2003
@@ -3,7 +3,7 @@
 dnl This just unconditionally sets the options.  It should offer an option for
 dnl explicitly giving the path to libraw1394 on the configure command line.
 dnl
-AC_DEFUN(AC_LIB_RAW1394_FLAGS, [
+AC_DEFUN([AC_LIB_RAW1394_FLAGS], [
 LIBRAW1394_CPPFLAGS=""
 LIBRAW1394_CFLAGS=""
 LIBRAW1394_LIBS="-lraw1394"
@@ -16,7 +16,7 @@
 dnl
 dnl AC_LIB_RAW1394_HEADERS([ACTION_IF_FOUND[,ACTION_IF_NOT_FOUND]])
 dnl
-AC_DEFUN(AC_LIB_RAW1394_HEADERS, [
+AC_DEFUN([AC_LIB_RAW1394_HEADERS], [
 AC_REQUIRE([AC_LIB_RAW1394_FLAGS])
 
 ac_libraw1394_save_cppflags=$CPPFLAGS
@@ -38,7 +38,7 @@
 dnl
 dnl AC_LIB_RAW1394_LIBVERSION(MINIMUMVERSION[,ACTION_IF_FOUND[,ACTION_IF_NOT_FOUND]])
 dnl
-AC_DEFUN(AC_LIB_RAW1394_LIBVERSION, [
+AC_DEFUN([AC_LIB_RAW1394_LIBVERSION], [
 AC_REQUIRE([AC_PROG_CC])
 AC_REQUIRE([AC_LIB_RAW1394_FLAGS])
 
@@ -72,7 +72,7 @@
 dnl
 dnl AC_LIB_RAW1394_RUNTEST(MINIMUMVERSION[,ACTION_IF_FOUND
 dnl                        [,ACTION_IF_NOT_FOUND[,ACTION_IF_CROSS_COMPILING]]])
-AC_DEFUN(AC_LIB_RAW1394_RUNTEST, [
+AC_DEFUN([AC_LIB_RAW1394_RUNTEST], [
 ac_libraw1394_save_cppflags=$CPPFLAGS
 ac_libraw1394_save_cflags=$CFLAGS
 ac_libraw1394_save_libs=$LIBS
@@ -135,7 +135,7 @@
 dnl Versions before 0.9 can't be checked, so this will always fail if the
 dnl installed libraw1394 is older than 0.9 as if the library weren't found.
 dnl
-AC_DEFUN(AC_LIB_RAW1394, [
+AC_DEFUN([AC_LIB_RAW1394], [
 
 AC_LIB_RAW1394_FLAGS
 AC_LIB_RAW1394_HEADERS(ac_libraw1394_found=yes, ac_libraw1394_found=no)
